manifest.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "Video Speed Controller",
   "short_name": "videospeed",
-  "version": "0.6.5",
-  "manifest_version": 2,
+  "version": "0.7.0",
+  "manifest_version": 3,
+  "minimum_chrome_version": "89",
```

Comment:
question why do we want to put the minimum chrome version on 89?
Why not keep it so that chrome users from 2020 could even install it.

Reply:
Chrome 89 is the first stable release to support Manifest V3.

Reply:
A thanks could not find it anywhere.